src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -150,6 +150,35 @@
  * <pre>
  * &lt;module name="JavadocMethod"/&gt;
  * </pre>
```

Comment:
Please add examples for all the properties of the check. https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/108a89b_2023172613/config_javadoc.html#JavadocMethod_Properties
Some are there, add for the rest.

Reply:
@Vyom-Yadav I have added the examples for the remaining properties and added their CLI test results to the PR description.